phoenicis-library/src/main/java/org/phoenicis/library/ShortcutManager.java

```diff
@@ -87,6 +88,16 @@ public void createShortcut(ShortcutDTO shortcutDTO) {
             } catch (IOException | IllegalArgumentException e) {
                 LOGGER.warn("Error while creating shortcut icon", e);
             }
+            try {
+                if (shortcutDTO.getCategoryIcon() != null) {
+                    File file = createFileWithFallback(shortcutDTO.getCategoryIcon(), "phoenicis.png");
+                    if (file.exists()) {
```

Comment:
Is it possible at all that this file does not exist? If this is the case should we maybe throw/show an error?

Reply:
Not quite sure. Normally not. I applied the same logic as for the shortcut icon though.